docs/ops.html

```diff
@@ -229,6 +229,63 @@ <h4><a id="basic_ops_consumer_group" href="#basic_ops_consumer_group">Managing C
   Deletion of requested consumer groups ('my-group', 'my-other-group') was successful.
   </pre>
 
+  <p>
+  To reset offsets of a consumer group to latest offset, first make sure the instances are inactive, then:
+
+  <pre class="brush: bash;">
+  &gt; bin/kafka-consumer-groups.sh --bootstrap-server localhost:9092 --reset-offsets --group consumergroup1 --topic topic1 --to-latest
+
+  TOPIC                          PARTITION  NEW-OFFSET
+  topic1                         0          0
+  </pre>
+
+  --reset-offsets supports one consumer group at the time.
+  <p>
+  It has 3 execution options:
+  <ul>
+    <li>
+      (default) to plan which offsets to reset.
```

Comment:
I know this is the same language used above, but it's unclear what we mean by "plan." Perhaps a better phrase would be "display"?

Reply:
Ack